code/modules/surgery/organs/internal/_internal_organ.dm

```diff
@@ -32,13 +32,13 @@
 		return
 
 	if(owner)
-		if(owner.bodytemperature > T0C)
-			var/air_temperature_factor = min((owner.bodytemperature - T0C) / T20C, 1)
+		if(owner.bodytemperature > BODYTEMP_CRYO_THRESHOLD)
+			var/air_temperature_factor = min((owner.bodytemperature - BODYTEMP_CRYO_THRESHOLD) / (T20C - BODYTEMP_CRYO_THRESHOLD), 1)
 			apply_organ_damage(decay_factor * maxHealth * seconds_per_tick * air_temperature_factor)
 	else
 		var/datum/gas_mixture/exposed_air = return_air()
-		if(exposed_air && exposed_air.temperature > T0C)
-			var/air_temperature_factor = min((exposed_air.temperature - T0C) / T20C, 1)
+		if(exposed_air && exposed_air.temperature > BODYTEMP_CRYO_THRESHOLD)
+			var/air_temperature_factor = min((exposed_air.temperature - BODYTEMP_CRYO_THRESHOLD) / (T20C - BODYTEMP_CRYO_THRESHOLD), 1)
```

Comment:
Don't change this.
Below 0 celsius, organs should start taking reduced damage up until -20 degrees
Also I'm pretty sure this will allow for organs to start healing at low enough temperatures.

It's intentional that organs are not supposed to take any damage at temperatures below 0 celsius, so this would really be a balance change if anything. It's supposed to be the case that people in space don't take any organ damage.